app/pages/[teamSlug]/customers/index.page.tsx

```diff
@@ -29,7 +29,6 @@ const CustomersPage = (): JSX.Element => {
     <ProtectedPage>
       <article>
         <div className="flex justify-between">
-          <h2>Customers</h2>
```

Comment:
I think we can remove the whole page. There is no link to this page anymore, right?